src/components/common/SearchFiltersWrapperV2/FeaturedFilters.tsx

```diff
@@ -14,7 +14,7 @@ interface FeaturedFiltersProps {
 type MultiSelectFilters = Record<string, (string | number | null)[]>;
 
 const defaultSearchNodeCreator = (field: string, values: (number | string | null)[]) => ({
-  field: 'status',
+  field,
```

Comment:
Interesting that `field` was being ignored previously!

Reply:
Yea that is a bit suspect... would have caught it sooner if it did!

Reply:
We would need to set `noUnusedParameters` to true in tsconfig.json.. https://www.typescriptlang.org/tsconfig#noUnusedParameters maybe something we can do later.
Tried it and we have a ton of such errors.